### PR TITLE
Code Review practice review branch

### PR DESCRIPTION
**Pull request by benardop**
This is a pull request created for the purposes of doing the first code review practice.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,10 @@
-!DOCTYPE html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="styles.css">
     <title>Microverse Product Landing Page</title>
   </head>
-     <header id="header">
```
Comment (benardop) [inline index.html:8]: Good work on your first HTML project. You may please check your page structure and follow the correct HTML page flow.  You may please remove the extra header element left on the page

**Review by benardop — COMMENTED** [review]
You may please add a detailed README file to your project. 
You may also get a copy of linters file from microverse and place them in your project directory so you can be able to check linters errors in your project before making a pull request.
Thanks.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @benardop , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- [ ] While you did a good job you still have to add at least one other inline comment.


## Highlights
- Good use of  Github tools

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    nav.navbar {
+      font-size: 80px;
+    }
+  }
+  body {
+    display:flex
+  }
+
+
+    @media (max-width: 600px) {
```
Comment (benardop) [inline styles.css:25]: Good work so far! 👍 
Please be sure to use the correct indentation and spacing in your styles.cc file.

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @benardop 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**